examples/test-app-e2e/cypress.config.ts

```diff
@@ -1,6 +1,21 @@
 import { nxE2EPreset } from '@nx/cypress/plugins/cypress-preset';
 import { defineConfig } from 'cypress';
 
+const nxe2eConfig = nxE2EPreset(__filename, { cypressDir: 'src' });
+
+// Adds logging to terminal for debugging
+// https://docs.cypress.io/api/commands/task#Usage
+// Usage: `cy.task('log', 'my message');
 export default defineConfig({
-  e2e: nxE2EPreset(__filename, { cypressDir: 'src' }),
+  e2e: {
+    ...nxe2eConfig,
+    setupNodeEvents(on) {
+      on('task', {
+        log(message) {
+          console.log(message);
```

Comment:
I'd hide this behind some env flag. I can clutter the output.

Reply:
This just make the `cy.task('log')` available. It won't log anything unless its called directly.